doc/19-technical-concepts.md

```diff
@@ -1514,6 +1514,76 @@ Message updates will be dropped when:
 * Notification does not exist.
 * Origin endpoint's zone is not allowed to access this checkable.
 
+#### event::UpdateLastNotifiedStatePerUser <a id="technical-concepts-json-rpc-messages-event-updatelastnotifiedstateofuser"></a>
```

Comment:
The url id no longer matches the cluster event name! (`event-updatelastnotifiedstateofuser`).